src/Components/Charts/StatusPageBarChart/index.jsx

```
@@ -1,211 +1,211 @@
+};
+
+Bar.propTypes = {
+	width: PropTypes.string.isRequired,
+	height: PropTypes.string.isRequired,
+	backgroundColor: PropTypes.string.isRequired,
+	borderRadius: PropTypes.string,
+	children: PropTypes.node,
+};
+
+/* TODO add prop validation and jsdocs */
+const StatusPageBarChart = ({ checks = [] }) => {
```

ESLint

[error] 'checks' is missing in props validation

(react/prop-types)